transfer-instructions.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: docs
+title: "Transfer Instructions"
+---
+
+## Transfer Overview & Setup
```

Comment:
I'm curious as to how accurately this page represents current practices. I'm not an expert but it seems like there's potentially a lot of information that's out of date and/or doesn't reflect current workflows and tools.